sb-cb-ui-consumption/library/sunbird-cb/consumption/src/lib/_common/cards/card-progress-portrait-lib/card-progress-portrait-lib.component.html

```diff
@@ -1,114 +1,120 @@
+  <div class="course_widget portrait_widget flex  margin-bottom-xs">
+    <div class="flex flex-end flex-middle course_port">
+      <mat-icon class="mat-icon main_icon  ws-mat-default-text margin-right-xs">video_library</mat-icon>
+      <sb-uic-display-content-type i18n-title title="Content Type" *ngIf="widgetData?.content?.primaryCategory"
+        class="ws-mat-black60-text font-normal mat-caption " [displayContentType]="widgetData?.content?.primaryCategory==='Course Unit'
+        ?'Module'
+        :widgetData?.content?.primaryCategory">
+      </sb-uic-display-content-type>
+    </div>
+  </div>
+  <div class="flex">
+    <div class="margin-right-s">
+      <ng-container *ngIf="widgetData?.content?.posterImage">
+        <div class=" img-block-portrait position-relative">
+          <img mat-card-image [src]="widgetData?.content?.posterImage | pipePublicURL" loading="lazy"
+          class="card-image ws-mat-primary-lite-background margin-remove" [wsUtilsDefaultThumbnail]="defaultThumbnail"
+          [alt]="widgetData?.content?.name" />
+          <div class="duration-box right-corner-portrait" *ngIf="widgetData?.content?.duration > 0">
+            <mat-icon class="mat-icon main_icon mr-1">access_time</mat-icon>
+            <span class=" ws-mat-white-text duration">
+              {{ (widgetData?.content?.duration || 120)| pipeDurationTransform: 'hms' }}
+            </span>
+          </div>
+          <div class="duration-box right-corner-portrait" *ngIf="widgetData?.content?.programDuration > 0">
+            <mat-icon class="mat-icon main_icon mr-1">access_time</mat-icon>
+            <span class=" ws-mat-white-text duration">
+              {{widgetData?.content?.programDuration}} {{widgetData?.content?.programDuration === 1 ? 'day' : 'days'}}
+            </span>
+          </div>
+          <ng-container *ngIf="cbPlanMapData && cbPlanMapData[widgetData?.content?.identifier]">
+                <ng-container [ngTemplateOutlet]="cbpPlan" [ngTemplateOutletContext]="{data: {
+                  cbpData: cbPlanMapData[widgetData?.content?.identifier]
+                }}"></ng-container>
+          </ng-container>
         </div>
+      </ng-container>
+      <ng-container *ngIf="!widgetData?.content?.posterImage">
+        <ng-container *ngIf="widgetData?.content?.appIcon; else defaultImg">
+          <img mat-card-image [src]="widgetData?.content?.appIcon | pipePublicURL" loading="lazy"
+          class="card-image ws-mat-primary-lite-background margin-remove"  [wsUtilsDefaultThumbnail]="defaultThumbnail"
+          [alt]="widgetData?.content?.name" />
+        </ng-container>
+        <ng-template #defaultImg>
+          <img mat-card-image [src]="defaultThumbnail" loading="lazy"
+          class="card-image ws-mat-primary-lite-background margin-remove"
+          [alt]="widgetData?.content?.name" />
+        </ng-template>
```

Comment:
Please confirm, all these changes should not impact anywhere in learner portal and wherever we are using it